test/e2e/util/util.go

```diff
@@ -353,6 +359,17 @@ func NewCertManagerBasicCertificateRequest(name, issuerName string, issuerKind s
 	}, sk, nil
 }
 
+func NewCertManagerBasicCertificateRequestWithUsages(name, issuerName string, issuerKind string, duration *metav1.Duration,
```

Comment:
We're moving away from creating these `util` functions in favour of the `gen` package which can be used to generate resources.

Can you remove this, and instead update the places that call it to call:

```
gen.CertificateFrom(cert, gen.SetCertificateKeyUsages(...))
```

Reply:
this is going to require a fair bit of code, but sure